library/init/meta/simp_tactic.lean

```diff
@@ -372,7 +372,7 @@ meta def to_simp_lemmas : simp_lemmas → list name → tactic simp_lemmas
 | S (n::ns) := do S' ← S^.add_simp n, to_simp_lemmas S' ns
 
 meta def mk_simp_attr (attr_name : name) : command :=
-do t ← to_expr ``(caching_user_attribute simp_lemmas),
+do let t := ```(caching_user_attribute simp_lemmas),
    v ← to_expr ``({name     := %%(quote attr_name),
```

Comment:
@rlewis1988 I believe you expressed slight concern about the old `to_expr` functions going away, any opinions about bringing them back? It would be a typeclass function of type `α → expr`, so no `tactic`.

@leodemoura I can't think of a general solution. Even if we also passed the reflected type to the function, it would still have to do some partial elaboration to infer universe parameters. In that case, the user may as well use `to_expr ∘ quote`.